velodyne_pointcloud/src/conversions/convert.cc

```diff
@@ -21,29 +21,40 @@ namespace velodyne_pointcloud
 {
   /** @brief Constructor. */
   Convert::Convert(ros::NodeHandle node, ros::NodeHandle private_nh):
-    data_(new velodyne_rawdata::RawData())
+    data_(new velodyne_rawdata::RawData()),
+    prev_angle_(0.0),
+    prev_stamp_(ros::Time())
   {
     data_->setup(private_nh);
 
 
     // advertise output point cloud (before subscribing to input data)
     output_ =
       node.advertise<sensor_msgs::PointCloud2>("velodyne_points", 10);
-      
+    
+    // advertise output point cloud (before subscribing to input data)
```

Comment:
This comment appears to be repeated from the previous block. I guess that was not the intent?

Reply:
Fixed.